spec/rspec/core/formatters/base_text_formatter_spec.rb

```diff
@@ -30,6 +35,12 @@
       formatter.close(RSpec::Core::Notifications::NullNotification)
       expect(output_to_close.closed?).to be(false)
     end
+
+    it "restores the output's sync setting" do
+      expect {
+        formatter.close(RSpec::Core::Notifications::NullNotification)
+      }.to change(output_to_close, :sync).from(true).to(false)
+    end
```

Comment:
I'm glad this passes as is, but is it in the right place? This is actually implemented in the base formatter, and if this is testing the `super` call then it doesn't cover the base formatter and the json formatter, prehaps some of these should be a shared example we call across all the formatters that need it?